### PR TITLE
Update Kyle Combs' Main Page

### PR DESCRIPTION
**Pull request by Yoshimitsu45**

### CODE REVIEW

File: users/combsky/index.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
 layout: default
 author: Kyle Combs
 ---
 Kyle Combs
 ================================
 
-Scribble
\ No newline at end of file
+Scribble
+Scribble2
```
Comment (albeanth) [inline users/combsky/index.md:9]: I think you need more scribbles.

Comment (Yoshimitsu45) [reply users/combsky/index.md:9]: I have enough scribbles, thank you very much.

Comment (grechasneak) [reply users/combsky/index.md:9]: 💯

**Comment by albeanth** [comment]
Ok, I like this now. :shipit:

File: users/combsky/index.md (hunk 1 above)
Comment (Aarontama) [reply users/combsky/index.md:9]: :shipit:

**Comment by harterj** [comment]
dj scrib :shipit:

**Pull request merged**